java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/KinesisProducerConfiguration.java

```diff
@@ -1439,6 +1479,46 @@ public KinesisProducerConfiguration setVerifyCertificate(boolean val) {
         return this;
     }
 
+    /**
+     * If you have users going through a proxy, set the host here.
```

Comment:
Can we add an actual example of how the hostName should be added like for example "http://proxy.example.com" etc which will be useful for customers

Reply:
done